numpy/random/_examples/numba/extending_distributions.py

```diff
@@ -35,19 +41,19 @@
     raise RuntimeError('Required DLL/so file was not found.')
 
 ffi.cdef("""
-double random_gauss_zig(void *bitgen_state);
+double random_standard_normal(void *bitgen_state);
 """)
 x = PCG64()
 xffi = x.cffi
 bit_generator = xffi.bit_generator
 
-random_gauss_zig = lib.random_gauss_zig
+random_standard_normal = lib.random_standard_normal
 
 
 def normals(n, bit_generator):
     out = np.empty(n)
     for i in range(n):
-        out[i] = random_gauss_zig(bit_generator)
+        out[i] = random_standard_normal(bit_generator)
     return out
 
 
```

Comment:
I wonder if we could run this as part of the tests, like we run the `extending.py` example

Reply:
Depends on numba or at least CFFI, so probably not worth it.  But definitely a risk that examples get stale, so ideally these things get run somewhere.